store_boltdb.go

```diff
@@ -60,7 +60,7 @@ func newBoltDBStore(location *url.URL) (storage, error) {
 }
 
 // Set adds a token to the store
-func (r *boltdbStore) Set(key, value string) error {
+func (r *boltdbStore) Set(key, value string, expiration time.Duration) error {
```

Comment:
@msuret don't you think we should use `expiration` here? 

From my limited knowledge it seems that BoltDB doesn’t have a way of pruning old data, wouldn't make sense to check the expiration time and remove it from BoltDB?

Reply:
Indeed I didn't find an easy way to expire BoltDB items.
Are you thinking about adding a background goroutine that periodically cleans Boltdb ?
That's a good idea, but I don't really have time to work on this right now.

Reply:
@msuret no problem at all. I will just check with others on the mailing list if we can create a follow up Jira. Thank you!